libs/agno/agno/workflow/step.py

```diff
@@ -60,6 +61,8 @@ class Step:
     # If False, only warn about missing inputs
     strict_input_validation: bool = False
 
+    add_workflow_history: bool = False
```

Comment:
Should we not have num_history_runs here as well?

Reply:
yes we should, added